### ISSUE TITLE
Bug in prediction code?

### ISSUE THREAD

**Issue by danielegrattarola** [VGraphRNN/VGRNN#5]
Hello, 

I am looking at the `VGRNN_prediction.py` script and I found what seems to be a bug on [this line](https://github.com/VGraphRNN/VGRNN/blob/352da1b63fe6ea1db18c7d512793a21e0d9a5bbb/VGRNN_prediction.py#L906):

```python
nll_loss += self._nll_bernoulli(dec_t_sl, adj_orig_dense_list[t])
```

Shouldn't it be: 

```python
nll_loss += self._nll_bernoulli(dec_t_sl, adj_orig_dense_list[t + 1])
```
in order to do link prediction for the graph at time T+1?
Did I misunderstand something?

Thanks

**Comment by Frankie123421** [comment]
I think it's actually the log-likelihood term of the loss function (equation 7), which means reconstructing A[t] by Z[t] at each time step.
